docs/tutorials/configure/pet-photographer.md

```diff
@@ -0,0 +1,267 @@
+
+1. Then, set **Hue Tolerance** to `0.06` and **Segment Size px** to `100`.
+
+Your configuration should look like the following:
+
+![The vision service configuration panel showing the color set to blue, the hue tolerance set to 0.06, and the segment size set to 100.](/tutorials/pet-photographer/vision-service.png)
+
+For more detailed information see [Configure a color detector](/services/vision/detection/#configure-a-color_detector).
+
+{{% /tab %}}
+{{% tab name="JSON Template" %}}
+
+Add the vision service object to the services array in your rover’s raw JSON configuration:
+
+```json {class="line-numbers linkable-line-numbers"}
+  {
+    "name": "my_color_detector",
+    "type": "vision",
+    "model": "color_detector",
+    "attributes": {
+      "segment_size_px": 100,
+      "detect_color": "#43a1d0",
+      "hue_tolerance_pct": 0.06
+    }
+  },
+  ... // Other services
+```
+
+{{% /tab %}}
+{{< /tabs >}}
+
+Click **Save Config** and head back to the **Builder** mode.
+
+### Data management service to collect images
+
+To enable data capture on your robot, add and configure the [data management service](/services/data/) to capture and store data on your robot's computer.
+
+{{< tabs >}}
+{{% tab name="Builder" %}}
+
+1. At the bottom-left of the page, click **Create service**.
+1. Choose `Data Management` as the type and name your instance of the data manager `dm`.
+   This service syncs data from your robot to the Viam app in the cloud.
+1. Select **Create**.
+1. On the panel that appears, you can manage the capturing and syncing functions individually.
+   By default, the data management service captures data every 0.1 minutes in the <file>~/.viam/capture</file> directory.
+
+   You can leave the default settings as they are.
+   Click **Save Config** at the bottom of the window.
+
+   ![An instance of the data management service named "dm". The cloud sync and capturing options are toggled on and the directory is empty. The interval is set to 0.1](/tutorials/pet-photographer/data-management-services.png)
+
+   For more detailed information see [Add the data management service](/services/data/configure-data-capture/#add-the-data-management-service).
+   {{% /tab %}}
+   {{% tab name="JSON Template" %}}
+   Add the vision service object to the services array in your rover’s raw JSON configuration:
+
+```json {class="line-numbers linkable-line-numbers"}
+    {
+      "name": "dm",
+      "type": "data_manager",
+      "namespace": "rdk",
+      "attributes": {
+        "sync_interval_mins": 0.1,
+        "capture_dir": "",
+        "tags": [],
+        "additional_sync_paths": []
+      }
+    },
+  ... // Vision and other services
+```
+
+Click **Save Config** and head back to the **Builder** mode.
+
+{{% /tab %}}
+{{< /tabs >}}
+
+## Configure the color filter camera
+
+Before you can interact with the vision and data management services, you must configure your camera to filter color and store photos to Viam's cloud.
```

Comment:
You can actually already interact with the vision service programmatically so let's change this slightly:

```suggestion
With the vision and data management services configured, you can now configure your camera to filter by color and store photos to Viam's cloud.
```